src/tests/utils/setup.ts

```diff
@@ -82,6 +82,34 @@ beforeAll(async () => {
   };
 }, TIMEOUT);
 
+beforeEach(async () => {
+  await global.db.query(`
+  DO $$
+  DECLARE 
+    tbl RECORD;
+    seq RECORD;
+  BEGIN
+    -- truncate all tables in database
+    FOR tbl IN 
+      SELECT * FROM information_schema.tables
+      WHERE table_name!= 'migrations'
+      AND table_schema='public'
+      AND table_type= 'BASE TABLE'
+    LOOP
+      EXECUTE format('TRUNCATE public.%s CASCADE', tbl.table_name);
+    END LOOP;
+
+    -- restart all sequences
+    FOR seq IN 
+      SELECT * FROM information_schema.sequences
+      WHERE sequence_schema= 'public'
+    LOOP
+      EXECUTE format('ALTER sequence %s RESTART WITH 1', seq.sequence_name);
+    END LOOP;
+  END $$ LANGUAGE plpgsql;   
```

Comment:
ideally this function should be stored in separate .sql file and imported here. Also some clarifying comments would be very helpful for understanding

Reply:
done